src/lib.rs

```diff
@@ -1195,6 +1254,13 @@ where
             StreamResult::Finished => Ok(StreamResult::Finished),
         }
     }
+
+    pub async fn cleanup(&mut self) {
```

Comment:
...Or manual cleanup isn't so bad either, as long as it's documented.

Reply:
Documented the cleanup steps.